File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/drm/model/DRMOperationParam.java
package org.ngbp.jsonrpc4jtestharness.rpc.drm.model;

import java.util.List;

public class DRMOperationParam {
    public String systemId;
    public String service;
    public List<Message> message;
    class Message{
        public String operation;
        public String licenseUri;
    }
}

File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/drm/model/DRMOperationParam.kt
package org.ngbp.jsonrpc4jtestharness.rpc.drm.model

data class DRMOperationParam (
    var systemId: String? = null,
    var service: String? = null,
    var message: MutableList<Message?>? = null
)

data class Message (
    var operation: String? = null,
    var licenseUri: String? = null
)
